api/src/controllers/podcast.js

```diff
@@ -13,16 +13,19 @@ import { PodcastQueueAdd, OgQueueAdd } from '../asyncTasks';
 
 exports.list = async (req, res) => {
 	let query = req.query || {};
-
 	let podcasts;
 
 	if (query.type === 'recommended') {
 		podcasts = await getPodcastRecommendations(req.User._id.toString(), 7);
+
+		if (!podcasts.length) {
+			return res.json([]);
+		}
```

Comment:
It looks as though getPodcastRecommendations already returns `[]` so you don't need this right? @nparsons08